src/.ocamlformat

```diff
@@ -0,0 +1,66 @@
+profile=ocamlformat
+quiet=false
+max-iters=10
+comment-check=true
+wrap-fun-args=true
+wrap-comments=false
+type-decl-indent=2
+type-decl=compact
+stritem-extension-indent=0
+space-around-variants=true
+space-around-records=true
+space-around-lists=true
+space-around-arrays=true
+single-case=compact
+sequence-style=separator
+sequence-blank-line=preserve-one
+parse-docstrings=false
```

Comment:
Making this `true` reformats docstrings pretty nicely.

I'll have to massage some of them where we had the pattern
```
(*
*
*
*)
```
because it treats the asterisks as text.

Reply:
Some our doccomments have explicit formatting, so they have to become vanilla comments to preserve that formatting. @mrmr1993 is that OK?

Reply:
I think that style is non-standard for OCaml anyway, seems more like a Java-ism (or a cousin like JS or C++). Can we lose the asterisks entirely?
A sample would be helpful, especially since I'm sure there'll be other opinions on these things.